### DISCUSSION TITLE
招募远程游戏开发者

### DISCUSSION THREAD

**Discussion by gonglei007** [gonglei007/GameDevMind#23] [General]
我们这有些游戏开发相关的有意思的事情（自由岗位/优质实践等），有兴趣的游戏开发者们可以详细沟通。
![招募海报 001](https://github.com/gonglei007/GameDevMind/assets/5304584/9cf6047b-42e4-418e-b5da-782d2d9a992d)
![招募海报 002](https://github.com/gonglei007/GameDevMind/assets/5304584/a008166b-9972-490e-bec8-f780e42fa8a2)
![招募海报 003](https://github.com/gonglei007/GameDevMind/assets/5304584/a2b6a703-ddad-45a0-9342-7f029acaee61)

**Comment by 120014027-oss** [comment]
awesome .!!

**Comment by 120014027-oss** [comment]
konichiwa sorry I don't speajkJapanese  lets be best friends for ever
